static/swagger-specs/destination-authoring.yaml

```diff
@@ -1662,6 +1662,16 @@ definitions:
             example: true
             title: "Is Identity Required"
             description: "Use true if you users should be able to map identity namespaces from Experience Platform to your desired schema."
+          segmentNamespaceAllowList:
+            type: "array"
+            example: ["AudienceManager"]
+            title: "Segment Namespace Allow List"
+            description: "Defines specific audience namespaces from which users can map audiences to the destination. Use this parameter to restrict Experience Platform users to export audiences from only the audience namespaces that you define in the array. This parameter cannot be used together with `segmentNamespaceDenyList`. <br><br> Example: `\"segmentNamespaceAllowList\": [\"AudienceManager\"]` will allow users to map only audiences from the `AudienceManager` namespace to this destination. <br><br> To allow users to export any audience to your destination, you can omit this parameter and set `\"segmentNamespaceDenyList\":[]`. If both `segmentNamespaceAllowList` and `segmentNamespaceDenyList` are missing from your configuration, users will only be able to export audiences originating from the [Segmentation Service](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home)."
+          segmentNamespaceDenyList:
+            type: "array"
+            example: ["AudienceManager"]
+            title: "Segment Namespace Deny List"
+            description: "Restricts users from mapping audiences to the destination, from the audience namespaces defined in the array.  <br><br>To allow the export of all audiences, regardless of the origin, set `\"segmentNamespaceDenyList\":[]`. This parameter cannot be used together with `segmentNamespaceAllowed`. <br><br>Example: `\"segmentNamespaceDenyList\": [\"AudienceManager\"]` will block users from mapping audiences from the `AudienceManager` namespace to this destination. <br><br>If both `segmentNamespaceAllowed` and `segmentNamespaceDenyList` are missing from your configuration, users will only be able to export audiences originating from the [Segmentation Service](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home)."
```

Comment:
Some comments: 

Rewording:
"Restricts users from mapping audiences from the audience namespaces defined in the array to the destination.

Let's lead with a very clear, bold statement that this is the recommended option. I would also probably add an example where we don't block any namespace in the example param. The message that we want to make very clear here is that there is no reason to restrict namespaces.